storage/storage-samples/storage-rv-sample/src/main/java/org/fido/iot/storage/To0DbStorage.java

```diff
@@ -98,6 +100,100 @@ public long storeRedirectBlob(Composite voucher, long requestedWait, byte[] sign
     throw new ResourceNotFoundException(guid.toString());
   }
 
+  @Override
+  public List<String> getOvKeysAllowlist() {
+
+    List<String> keysInAllowList = new ArrayList<>();
+
+    String sql = "SELECT PUBLIC_KEY_HASH FROM OV_KEYS_ALLOWLIST";
+
+    try (Connection conn = dataSource.getConnection();
+        PreparedStatement pstmt = conn.prepareStatement(sql)) {
+
+      try (ResultSet rs = pstmt.executeQuery()) {
+        while (rs.next()) {
+          keysInAllowList.add(rs.getString(1));
+        }
+      }
+
+    } catch (SQLException e) {
+      throw new RuntimeException(e);
+    }
+    return keysInAllowList;
+  }
+
+  @Override
+  public void setOvKeysAllowlist() {
+
+    List<String> allowlist = new ArrayList<>();
+    allowlist.add("42110E8F0F3184A1A5C51868BCBFF7144D66E41D1A188103C0264D5DA8BBCF88");
+    allowlist.add("707B6451B8319C28E412F847E17BB87995441AF356007A03A3A4AC7745A5223D");
+    allowlist.add("25D42F0536CE584E5812AB8750E80E7464742B4B65347BEA90AD4BBC71D3FFA6");
+    allowlist.add("283ADF4CCB527C19A72CFB21A9FF7B555788E6B365CEF3A26C6B876EE0FFE017");
+    allowlist.add("85A481BBC2DA15EDD7301FF92BA2BB60093D5864A8207F9D78A399B32AB4CFF4");
+    allowlist.add("31726603CB0751BFB926B6436369265557855744338FFC3307693E0D14D5241D");
+    allowlist.add("2ED65928AD50CB8542E648B9CD5C8B4BFB76DA870C723B16464F49F5140F7098");
+    allowlist.add("1DAC184C6A8BB2D00665F4CFC55B1F55AC9BFB4C899B06827C0C1990A1A0F74C");
+    allowlist.add("834F83875910C8507CE935BE2F947DCF854E6554C3ACB79893ACF91220EA5D8B");
+    allowlist.add("B4E95FB7062303BEB84FBB606ED75CCE99D1C4B6CC88F71E65286CAD7C74F3A5");
+
+    String sql = "INSERT INTO OV_KEYS_ALLOWLIST (PUBLIC_KEY_HASH) VALUES (?);";
```

Comment:
will this statement only be run when the table is empty? What happens when the pri component is re-run with an existing database?

Reply:
By re-run you mean application is restarted or onboarding process is done for another device?

Reply:
When the application is restarted. If the db gets recreated on restart then what happens to any entries the user may have inserted?

Reply:
Just saw the other comment indicating that the db contents are persistent between application restarts.